tensorflow_addons/callbacks/tests/avg_model_checkpoint_test.py

```diff
@@ -0,0 +1,206 @@
+import pytest
+import tensorflow as tf
+from tensorflow import keras
+from tensorflow_addons.callbacks import AverageModelCheckpoint
+from tensorflow_addons.optimizers import MovingAverage
+import os
+
+try:
+    import h5py
+except ImportError:
+    h5py = None
```

Comment:
Please add a message in the exception

Reply:
Actually I believe we can remove this import as it's not used in the file.